### ISSUE TITLE
Duplicate Preprocess on rollup.config.js

### ISSUE THREAD

**Issue by rraihansaputra** [lamualfa/setup-routify-ts#6]
https://github.com/lamualfa/routify-ts/blob/a5964b88fd0498d6e84d52379191049d01711b9b/helper/setupTypescript.js#L68

I know on the comments you indicated that finding the preprocess section is hard, but currently the `npx routify-ts` script outputs duplicate `preprocess` keys.

**Comment by lamualfa** [comment]
Can you provide me how to get that output? I just tried `init` and `convert` commands but couldn't find the output of duplicate `preprocess` keys.

I think there is something wrong with your `rollup.config.js` settings. Have you changed it after installing using `npx`?

**Comment by rraihansaputra** [comment]
Just ran it again, straight from `npx routify-ts <project-name>` with no options. No `yarn` or `npm i` executed yet.

```
rollup.config.js
...
    plugins: [
        svelte({
            dev: !production, // run-time checks      
            // Extract component CSS — better performance
            css: css => css.write(`bundle.css`),
		preprocess: sveltePreprocess(),
            hot: isNollup,
            preprocess: [
                autoPreprocess({
                    postcss: { plugins: [postcssImport()] },
                    defaults: { style: 'postcss' }
                })
            ]
        }),
...
```

There's two preprocess keys with different indentations. Attaching the complete result (with `node_modules` deleted) below.
[test-dup.zip](https://github.com/lamualfa/routify-ts/files/5660605/test-dup.zip)

**Comment by lamualfa** [comment]
@rraihansaputra Yeah, Good Catch. Thank you very much. I've found the problem. I will fix it now.

**Comment by lamualfa** [comment]
Ok, I have fixed the problem in the `v2.0.2` release. Thank you very much for the report @rraihansaputra.
